gene_finder.py

```diff
@@ -30,10 +30,17 @@ def get_complement(nucleotide):
     >>> get_complement('C')
     'G'
     """
-    # TODO: implement this
-    pass
+    if nucleotide == 'A': return 'T'
+    elif nucleotide == 'T': return 'A'
+    elif nucleotide == 'C': return 'G'
+    elif nucleotide == 'G': return 'C'
 
 
+# print(get_complement('A'))
```

Comment:
Remember to delete your intermediate print statements when submitting your code.